src/applications/my-education-benefits/config/form.js

```diff
@@ -85,7 +89,7 @@ const formFields = {
   phoneNumber: 'phoneNumber',
   mobilePhoneNumber: 'mobilePhoneNumber',
   viewBenefitSelection: 'view:benefitSelection',
-  benefitSelection: 'benefitSelection',
+  benefitRelinquished: 'benefitRelinquished',
```

Comment:
The name gave impression that it listed benefits selected to apply for, when in fact, it referred to relinquished benefits. Changed name to match.